src/test/java/utils/FileEncryptorTest.java

```diff
@@ -34,6 +36,28 @@ void decryptFileSourceFileNonExist() {
                 Paths.get("C:\\Users\\Yossef\\Katri\\IdeaProjects\\encryptor\\src\\main\\java\\outputFiles\\input1.txt")));
     }
 
+    @Test
+    void decryptFileDifferentNumberOfKeys(){
+        Path keyPath = Paths.get("C:\\InvalidEncryptionKeyException");
```

Comment:
Use relative path to the project directory

Reply:
done